### DISCUSSION TITLE
Underscore prefix for lodash functions and private variables

### DISCUSSION THREAD

**Discussion by yukinoda** [monstar-lab-oss/admin-panel-template-reactjs#85] [General] [answered]
I often see underscore prefix for private variables, won't be confusing having underscore prefix for lodash functions like below?
https://github.com/monstar-lab-oss/admin-panel-template-reactjs/blob/8139f0da8368cae5e5532124fdb62fb0ac95b527/src/components/atoms/DebouncedInput/DebouncedInput.tsx#L2

The underscore prefix helps me to note that is a lodash function but feel it may confuse user that is a private variable 🤔

**Comment by yukinoda** [comment]
something agreed
don't co-exist `_ prefix` private variables and `_ prefix` lodash functions

**Comment by yukinoda** [comment]
som lodash info

![image](https://user-images.githubusercontent.com/16083322/123057926-3f9bbb00-d443-11eb-90dd-a8ae2947b753.png)

https://itnext.io/lodash-es-vs-individual-lodash-utilities-size-comparison-676f14b07568

**Comment by geekonthemove** [comment, marked as answer]
I think for now it's fine to keep using `_` when importing _lodash_ functions, as is doesn't seem that any of us use `_` to mark private variables and functions.

But I also agree that if we were to use `_` for private variables and functions, then we shouldn't also use it for _lodash_. Then we would have to come up with a different solution.
